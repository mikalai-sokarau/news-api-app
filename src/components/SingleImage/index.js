import React from 'react';

const SingleImage = ({ url, fullSizeUrl, tags, user }) => {
  return (
    <div className="col s4 hoverable">
      <div className="card small">
        <div className="card-image">
          <a href={fullSizeUrl} target="_blank">
            <img src={url} alt={tags} />
          </a>
          <span className="card-title">{user}</span>
        </div>
        <div className="card-content">
          <span className="teal-text">Tags:</span>
          <p> {tags}</p>
        </div>
      </div>
    </div>
  );
};

export default SingleImage;
