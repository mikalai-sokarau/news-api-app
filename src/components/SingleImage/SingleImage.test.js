import React from 'react';
import { shallow } from 'enzyme';
import SingleImage from '../SingleImage';

describe('<SingleImage/>', () => {
  test('render SingleImage', () => {
    const wrapper = shallow(<SingleImage/>);
    expect(wrapper).toMatchSnapshot();
  });
});
