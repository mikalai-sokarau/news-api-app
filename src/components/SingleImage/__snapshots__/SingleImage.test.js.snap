// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SingleImage/> render SingleImage 1`] = `
<div
  className="col s4"
>
  <div
    className="card small hoverable"
  >
    <div
      className="card-image"
    >
      <a
        href=""
        target="_blank"
      >
        <img
          alt=""
          src=""
        />
      </a>
      <span
        className="card-title"
      />
    </div>
    <div
      className="card-content"
    >
      <span
        className="teal-text"
      >
        Tags:
      </span>
      <p>
         
      </p>
    </div>
  </div>
</div>
`;
